src/benchmark_api_internal.h

```diff
@@ -1,6 +1,8 @@
 #ifndef BENCHMARK_API_INTERNAL_H
 #define BENCHMARK_API_INTERNAL_H
 
+#include <array>
```

Comment:
still seeing these two headers. i'm sure they don't need to be added and i'd rather minimise any dependencies.